backend/src/api/Data/KeycloakJwtOptions.cs

```diff
@@ -1,5 +1,5 @@
 ﻿namespace api.Data;
-public class JwtOptions
+public class KeycloakJwtOptions
 {
     public string? Issuer { get; set; }
```

Comment:
I would sugest refactoring it into record or marking the fields as not nullable and required, as the app can't function without them anyway

Reply:
Fixed, I set the fields as not nullable and required.